docs/resources/control-flow/user-interactivity/forms/form-widgets/dropdown.md

```diff
@@ -241,7 +241,7 @@ You might need to disable a dropdown when certain conditions are not yet met or
 To disable the dropdown:
 
 1. Select the **DropDown** widget, move to the **Properties Panel > DropDown Search >** enable **Disable Dropdown** option.
-2. Click on **Unset** and select the source that returns the boolean value (i.e., True or False), such as boolean variable, [Conditions](../../../functions/control-flow-actions/conditional-logic), [Code Expression](#).
+2. Click on **Unset** and select the source that returns the boolean value (i.e., True or False), such as boolean variable, [Conditions](../../../../../resources/functions/conditional-logic), [Code Expression](../../../../../resources/functions/utility/#code-expressions).
```

Comment:
There should be a / before # in header references, this will throw an error in IDE.